src/encoded/static/components/browse/PublicationSearchView.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import React from 'react';
+import memoize from 'memoize-one';
+import _ from 'underscore';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['_' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4551745)